triggers/incident-triggered/README.md

```diff
@@ -0,0 +1,52 @@
+# victorops-trigger-incident-triggered
```

Comment:
You may want to call this something like `incident-escalated` so people don't get it confused with the custom webhook, which seems to have a completely different format. Or at least link to https://help.victorops.com/knowledge-base/escalation-webhooks/ somewhere.

Reply:
the custom one is more generic and doesn't necessarily trigger on incident creation/escalation - good call on the link though, added